### PR TITLE
Feature/42921

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "type": "magento2-theme",
     "require": {
       "magento/framework": "*",
-      "snowdog/module-alpaca-components": "~0.8.0",
+      "snowdog/module-alpaca-components": "dev-feature/42921",
```
Comment (Aekal) [inline composer.json]: Please change it to newer components version after release

File: styles/_theme.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Theme partials
+// // Theme partials
```
Comment (Aekal) [inline styles/_theme.scss]: Extra comment

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 - Product review details page
 - Moved theme styles to components
+- Styles and markup for chackout page
```
[the remark this answers is not in the document]
Comment (szafran89) [reply CHANGELOG.md]: fixed

File: Magento_Checkout/layout/checkout_index_index.xml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
-<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="checkout"
-      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd"
+<page
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="checkout"
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/layout/checkout_index_index.xml]: changed

File: Magento_Checkout/styles/_variables.scss [hunk 5]
```diff
@@ -165,19 +52,91 @@ $checkout-step-title__font-weight: $font-weight__light !default;
...
-//  Colors nesting
-$page__background-color: $color-white !default;
-
-//
-//  Variables missing in extend.scss
-//  ---------------------------------------------
-
-$addto-color: $primary__color !default;
-$addto-hover-color: $link__hover__color !default;
+// Alpaca variaables
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/_variables.scss]: Fixed

File: Magento_Checkout/styles/_variables.scss [hunk 6]
```diff
@@ -165,19 +52,91 @@ $checkout-step-title__font-weight: $font-weight__light !default;
...
+$font-weight-normal         : normal !default;
+$font-weight-bold           : bold !default;
+$font-weight-base           : $font-weight-normal !default;
+$font-line-height           : 1.5 !default;
+
+$font-size-extra-large      : 22px !default;
+$font-size-large            : 18px !default;
+$font-size-base             : 14px !default;
+$font-size-small            : 12px !default;
+$font-size-extra-small      : 10px !default;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/_variables.scss]: as we discussed over chat

File: Magento_Checkout/styles/blocks/_popups.scss [hunk 7]
```diff
@@ -1,69 +1,12 @@
...
 .popup-pointer {
-    @include lib-pointer(
-        $_size                       : $popup-marker__size,
-        $_background-color           : $popup__background,
-        $_border-color               : $popup__border-color,
-        $_position__vertical__value  : 2px,
-        $_position__horizontal__value: 2px
-    );
-    margin-left: -$popup-marker__size * 2;
+    margin-left: -7px * 2;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/blocks/_popups.scss]: changed

File: Magento_Checkout/styles/module/checkout/_authentication.scss [hunk 8]
```diff
@@ -1,16 +1,7 @@
...
 $block-auth__dropdown__padding         : $indent__m !default;
-$block-auth__dropdown__background-color: $color-white !default;
 $block-auth__or-label__size            : 36px !default;
 $block-auth__width                     : 0 !default;
-$block-auth__border                    : 1px solid $color-gray-light3 !default;
-
-//
-//  Common
-//  _____________________________________________
+$block-auth__border                    : 1px solid #c9c9c9 !default;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_authentication.scss]: changed

File: Magento_Checkout/styles/module/checkout/_authentication.scss [hunk 9]
```diff
@@ -112,16 +77,13 @@ $block-auth__border                    : 1px solid $color-gray-light3 !default;
...
-//  _____________________________________________
-
 @include min-screen($screen__m) {
     .authentication-dropdown {
         position: absolute;
         top: 100%;
         width: 100%;
-        background-color: $block-auth__dropdown__background-color;
+        margin-top: 10px;
+        background-color: #f7f7f9;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_authentication.scss]: changed

File: Magento_Checkout/styles/module/checkout/_estimated-total.scss [hunk 10]
```diff
@@ -1,42 +1,44 @@
...
+    }
+
+    &__counter {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-left: 3px;
+        border-radius: 5px;
+        background-color: #ff5501;
+        color: #fff;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_estimated-total.scss]: changed

File: Magento_Checkout/styles/module/checkout/_modals.scss [hunk 11]
```diff
@@ -1,51 +1,60 @@
...
+            @extend .abs-checkout-tooltip-content-position-top;
         }
     }
-}
 
-//
-//  Desktop
-//  _____________________________________________
+    .modal-inner-wrap {
+        background-color: #f7f7f9;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_modals.scss]: changed

File: Magento_Checkout/styles/module/checkout/_modals.scss [hunk 12]
```diff
@@ -1,51 +1,60 @@
...
 
-@include min-screen($screen__m) {
-    .checkout-index-index {
-        .modal-popup {
-            .form-shipping-address {
-                max-width: $checkout-shipping-address__max-width;
-            }
+        @include min-screen($screen__l) {
+            margin-left: (-800px / 2);
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_modals.scss]: changed

File: Magento_Checkout/styles/module/checkout/_progress-bar.scss [hunk 13]
```diff
@@ -1,53 +1,39 @@
...
-$checkout-progress-bar-item-element-inner__color              : $checkout-progress-bar-item__color !default;
 $checkout-progress-bar-item-element-inner__width              : $checkout-progress-bar-item-element__width - ($checkout-progress-bar-item-element-outer-radius__width * 2) !default;
 $checkout-progress-bar-item-element-inner__height             : $checkout-progress-bar-item-element-inner__width !default;
-$checkout-progress-bar-item-element-inner__active__content    : $icon-checkmark !default;
-$checkout-progress-bar-item-element-inner__active__font-size  : 28px !default;
 $checkout-progress-bar-item-element-inner__active__line-height: 1 !default;
 
+$green         : #31e37d !default;
+$white         : #fff !default;
+$color-primary : $green !default;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_progress-bar.scss]: removed

File: Magento_Checkout/styles/module/checkout/_progress-bar.scss [hunk 14]
```diff
@@ -1,53 +1,39 @@
...
 }
 
 .opc-progress-bar-item {
-    margin: 0 0 $checkout-progress-bar-item__margin;
     width: 50%;
     display: inline-block;
     position: relative;
     text-align: center;
     vertical-align: top;
+    font-size: 18px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_progress-bar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_progress-bar.scss [hunk 15]
```diff
@@ -73,19 +59,13 @@ $checkout-progress-bar-item-element-inner__active__line-height: 1 !default;
...
-            $_color      : $checkout-progress-bar-item__background-color,
-            $_font-family: inherit,
-            $_font-size  : $checkout-progress-bar__font-size,
-            $_font-style : inherit,
-            $_font-weight: $checkout-progress-bar__font-weight,
-            $_line-height: inherit
-        );
+        color: #e4e4e4;
+        font-weight: 300;
+        font-size: 14px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_progress-bar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_progress-bar.scss [hunk 16]
```diff
@@ -97,41 +77,29 @@ $checkout-progress-bar-item-element-inner__active__line-height: 1 !default;
...
 
     &._active {
         &:before {
-            background: $checkout-progress-bar-item__active__background-color;
+            background: $color-primary;
         }
 
         & > span {
-            color: $checkout-progress-bar-item__color;
+            color: #333;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_progress-bar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_progress-bar.scss [hunk 17]
```diff
@@ -140,24 +108,7 @@ $checkout-progress-bar-item-element-inner__active__line-height: 1 !default;
...
         & > span {
-            color: $checkout-progress-bar-item__color;
-
-            &:after {
-                content: $checkout-progress-bar-item-element-inner__active__content;
-                font-family: $icons__font-name;
-                line-height: $checkout-progress-bar-item-element-inner__active__line-height;
-                font-size: $checkout-progress-bar-item-element-inner__active__font-size;
-            }
+            color: #333;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_progress-bar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_shipping.scss [hunk 18]
```diff
@@ -76,27 +69,22 @@ $checkout-shipping-method__padding             : $indent__base !default;
...
                 @extend .abs-no-display-s;
                 visibility: hidden;
             }
         }
+
+        &__icon {
+            position: absolute;
+            top: 5px;
+            right: 5px;
+            fill: #31e37d;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_shipping.scss]: changed

File: Magento_Checkout/styles/module/checkout/_sidebar.scss [hunk 19]
```diff
@@ -1,18 +1,77 @@
...
+.opc-sidebar {
+    @include min-screen($screen__m) {
+        margin: 27px 0 20px;
         @include lib-layout-column(2, 2, $checkout-sidebar__columns);
     }
 }
+
+.checkout-summary {
+    padding: 15px;
+    background-color: #f7f7f9;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_sidebar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_sidebar.scss [hunk 20]
```diff
@@ -1,18 +1,77 @@
...
+            margin: 0 0 16px 0;
+        }
+    }
+}
+
+.custom-slide {
+    left: 0;
+
+    .modal-inner-wrap {
+        background-color: #f7f7f9;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_sidebar.scss]: changed

File: Magento_Checkout/styles/module/checkout/_sidebar.scss [hunk 21]
```diff
@@ -1,18 +1,77 @@
...
+        justify-content: space-between;
+    }
+
+    &__title-text {
+        align-self: center;
+        font-weight: 700;
+    }
+
+    &__content {
+        line-height: 1.5;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/module/checkout/_sidebar.scss]: changed

File: Magento_Checkout/web/template/payment-methods/list.html [hunk 22]
```diff
@@ -0,0 +1,28 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+ -->
+<div if="isPaymentMethodsAvailable()"
+     class="items payment-methods">
+    <div repeat="foreach: paymentGroupsList, item: '$group'"
+         class="payment-group">
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/payment-methods/list.html]: done

File: Magento_Checkout/web/template/payment-methods/list.html [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="divider">
+            <span class="divider__left"></span>
+            <span class="divider__right"></span>
+        </div>
+        <each args="data: getRegion($group().displayArea), as: 'method'" render=""/>
+    </div>
+</div>
+<div ifnot="isPaymentMethodsAvailable()"
+     class="no-payments-block"
+     translate="'No Payment Methods'">
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/payment-methods/list.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 24]
```diff
@@ -0,0 +1,112 @@
+<li
+    id="shipping"
+    class="checkout-shipping-address"
+    data-bind="fadeVisible: visible()"
+>
+    <div
+        class="step-title heading"
+        translate="'Shipping Address'"
+        data-role="title"
+    />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/shipping-address/shipping-method-item.html [hunk 25]
```diff
@@ -0,0 +1,44 @@
...
+        </div>
+    </td>
+    <td class="col col-price">
+        <each args="element.getRegion('price')" render="" />
+    </td>
+    <td
+        class="col col-method"
+        attr="'id': 'label_method_' + method.method_code + '_' + method.carrier_code"
+        text="method.method_title"
+    />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping-address/shipping-method-item.html]: done

File: Magento_Checkout/web/template/shipping-address/shipping-method-item.html [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+    <td
+        class="col col-carrier"
+        attr="'id': 'label_carrier_' + method.method_code + '_' + method.carrier_code"
+        text="method.carrier_title"
+    />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping-address/shipping-method-item.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 27]
```diff
@@ -0,0 +1,112 @@
...
+        <each args="getRegion('address-list-additional-addresses')" render="" />
+        <!-- Address form pop up -->
+        <if args="!isFormInline">
+            <button
+                type="button"
+                class="button button--secondary action-show-popup"
+                click="showFormPopUp"
+                visible="!isNewAddressAdded()"
+            >
+                <span translate="'New Address'" />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 28]
```diff
@@ -0,0 +1,112 @@
...
+            </button>
+            <div
+                id="opc-new-shipping-address"
+                visible="isFormPopUpVisible()"
+                render="shippingFormTemplate"
+            />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done.

File: Magento_Checkout/web/template/shipping.html [hunk 29]
```diff
@@ -0,0 +1,112 @@
...
+    class="checkout-shipping-method"
+    data-bind="fadeVisible: visible(), blockLoader: isLoading"
+    role="presentation"
+>
+    <div class="checkout-shipping-method">
+        <div
+            class="step-title heading"
+            translate="'Shipping Methods'"
+            data-role="title"
+        />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 30]
```diff
@@ -0,0 +1,112 @@
...
+                            <span translate="'Next'" />
+                        </button>
+                    </div>
+                </div>
+            </form>
+            <div
+                class="no-quotes-block"
+                ifnot="rates().length > 0"
+                translate="'Sorry, no quotes are available for this order at this time'"
+            />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 31]
```diff
@@ -0,0 +1,112 @@
...
+                    class="actions-toolbar"
+                    id="shipping-method-buttons-container"
+                >
+                    <div class="primary">
+                        <button
+                            data-role="opc-continue"
+                            type="submit"
+                            class="button continue"
+                        >
+                            <span translate="'Next'" />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/shipping.html [hunk 32]
```diff
@@ -0,0 +1,112 @@
...
+                <render args="shippingMethodListTemplate"/>
+                <div id="onepage-checkout-shipping-method-additional-load">
+                    <each args="getRegion('shippingAdditional')" render="" />
+                </div>
+                <div
+                    role="alert"
+                    if="errorValidationMessage().length"
+                    class="message notice"
+                >
+                    <span text="errorValidationMessage()" />
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/shipping.html]: done

File: Magento_Checkout/web/template/summary/item/details.html [hunk 33]
```diff
@@ -0,0 +1,55 @@
...
+            <span data-role="title" class="toggle">
+                <!-- ko i18n: 'View Details' --><!-- /ko -->
+            </span>
+            <div data-role="content" class="content">
+                <dl
+                    class="
+                        item-options
+                        list
+                        list--description
+                        minicart-product__attributes-list"
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/summary/item/details.html]: done

File: Magento_Checkout/web/template/summary/item/details/subtotal.html [hunk 34]
```diff
@@ -0,0 +1,7 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/summary/item/details/subtotal.html]: done

File: Magento_Checkout/web/template/summary/item/details/thumbnail.html [hunk 35]
```diff
@@ -0,0 +1,13 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/summary/item/details/thumbnail.html]: done

File: Magento_Checkout/web/template/summary/item/details/thumbnail.html [hunk 36]
```diff
@@ -0,0 +1,13 @@
...
+<span class="product-image-container"
+      data-bind="attr: {'style': 'height: ' + getHeight($parents[1]) + 'px; width: ' + getWidth($parents[1]) + 'px;' }">
+    <span class="product-image-wrapper">
+        <img
+            data-bind="attr: {'src': getSrc($parents[1]), 'width': getWidth($parents[1]), 'height': getHeight($parents[1]), 'alt': getAlt($parents[1]) }"/>
+    </span>
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/web/template/summary/item/details/thumbnail.html]: done

File: Magento_Tax/web/template/checkout/summary/tax.html [hunk 37]
```diff
@@ -0,0 +1,61 @@
...
+            <!-- /ko -->
+            <!-- ko if: !percent -->
+                <th class="mark" scope="row" data-bind="text: title"></th>
+            <!-- /ko -->
+            <!-- ko if: $index() == 0 -->
+                <td class="amount">
+                    <!-- ko if: $parents[1].isCalculated() -->
+                    <span class="price"
+                          data-bind="text: $parents[1].formatPrice($parents[0].amount), attr: {'data-th': title, 'rowspan': $parents[0].rates.length }"></span>
+                    <!-- /ko -->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/tax.html]: done

File: Magento_Tax/web/template/checkout/summary/tax.html [hunk 38]
```diff
@@ -0,0 +1,61 @@
...
+<!-- /ko -->
+<!-- ko if: ifShowValue() && ifShowDetails() -->
+    <tr class="totals-tax-summary"
+        data-bind="mageInit: {'toggleAdvanced':{'selectorsToggleClass': 'shown', 'baseToggleClass': 'expanded', 'toggleContainers': '.totals-tax-details'}}">
+        <th data-bind="text: title" class="mark" scope="row"></th>
+        <td data-bind="attr: {'data-th': title }" class="amount">
+            <!-- ko if: isCalculated() -->
+            <span class="price"
+                  data-bind="text: getValue()"></span>
+            <!-- /ko -->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/tax.html]: dane

File: Magento_Ui/web/templates/modal/modal-custom.html [hunk 39]
```diff
@@ -0,0 +1,61 @@
...
+                        <span><%= button.text %></span>
+                    </button>
+                <% }); %>
+            </footer>
+        <% } %>
+    </div>
+    <div data-role="focusable-end" tabindex="0"></div>
+</aside>
+
+
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Ui/web/templates/modal/modal-custom.html]: done

File: composer.json (hunk 1 above)
Comment (szafran89) [reply composer.json]: ok

File: styles/_theme.scss (hunk 2 above)
Comment (szafran89) [reply styles/_theme.scss]: done

File: Magento_Tax/web/template/checkout/summary/grand-total.html [hunk 40]
```diff
@@ -0,0 +1,38 @@
+<!-- ko if: isTaxDisplayedInGrandTotal && isDisplayed() -->
+    <div class="grand totals incl cart-totals__row cart-totals__row--summary">
+        <div class="mark cart-totals__row-label" scope="row">
+            <strong data-bind="i18n: inclTaxLabel"></strong>
+        </div>
+        <div data-bind="attr: {'data-th': inclTaxLabel}" class="amount cart-totals__row-value">
+            <strong><span class="price" data-bind="text: getValue()"></span></strong>
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/grand-total.html]: changed

File: Magento_Tax/web/template/checkout/summary/grand-total.html [hunk 41]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+    </div>
+    <div class="grand totals excl cart-totals__row cart-totals__row--summary">
+        <div class="mark cart-totals__row-label" scope="row">
+            <strong data-bind="i18n: exclTaxLabel"></strong>
+        </div>
+        <div data-bind="attr: {'data-th': exclTaxLabel}" class="amount cart-totals__row-value">
+            <strong><span class="price" data-bind="text: getGrandTotalExclTax()"></span></strong>
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/grand-total.html]: ok

File: Magento_Tax/web/template/checkout/summary/grand-total.html [hunk 42]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+    </div>
+<!-- /ko -->
+<!-- ko if: !isTaxDisplayedInGrandTotal && isDisplayed() -->
+    <div class="grand totals cart-totals__row cart-totals__row--summary">
+        <div class="mark cart-totals__row-label" scope="row">
+            <span data-bind="i18n: title"></span>
+        </div>
+        <div data-bind="attr: {'data-th': title}" class="amount cart-totals__row-value">
+            <span>
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/grand-total.html]: ok

File: Magento_Tax/web/template/checkout/summary/tax.html [hunk 43]
```diff
@@ -0,0 +1,61 @@
...
+            <!-- ko ifnot: isCalculated() -->
+                <span class="not-calculated" data-bind="text: getValue()"></span>
+            <!-- /ko -->
+        </div>
+    </div>
+<!-- /ko -->
+<!-- ko if: ifShowValue() && ifShowDetails() -->
+    <tr class="totals-tax-summary"
+        data-bind="mageInit: {'toggleAdvanced':{'selectorsToggleClass': 'shown', 'baseToggleClass': 'expanded', 'toggleContainers': '.totals-tax-details'}}">
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/tax.html]: ok

File: Magento_Tax/web/template/checkout/summary/tax.html [hunk 44]
```diff
@@ -0,0 +1,61 @@
...
+        <th data-bind="text: title" class="mark" scope="row"></th>
+        <td data-bind="attr: {'data-th': title }" class="amount">
+            <!-- ko if: isCalculated() -->
+            <span class="price"
+                  data-bind="text: getValue()"></span>
+            <!-- /ko -->
+            <!-- ko ifnot: isCalculated() -->
+            <span class="not-calculated"
+                  data-bind="text: getValue()"></span>
+            <!-- /ko -->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/tax.html]: done

File: Magento_Tax/web/template/checkout/summary/tax.html [hunk 45]
```diff
@@ -0,0 +1,61 @@
...
+            <!-- ko if: $index() == 0 -->
+                <td class="amount">
+                    <!-- ko if: $parents[1].isCalculated() -->
+                    <span class="price"
+                          data-bind="text: $parents[1].formatPrice($parents[0].amount), attr: {'data-th': title, 'rowspan': $parents[0].rates.length }"></span>
+                    <!-- /ko -->
+                    <!-- ko ifnot: $parents[1].isCalculated() -->
+                    <span class="not-calculated"
+                          data-bind="text: $parents[1].formatPrice($parents[0].amount), attr: {'data-th': title, 'rowspan': $parents[0].rates.length }"></span>
+                    <!-- /ko -->
```
[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Tax/web/template/checkout/summary/tax.html]: done

File: Magento_Checkout/styles/blocks/_popups.scss (hunk 7 above)
[the remark this answers is not in the document]
Comment (Aekal) [reply Magento_Checkout/styles/blocks/_popups.scss]: Not yet ;)

[the remark this answers is not in the document]
Comment (szafran89) [reply Magento_Checkout/styles/blocks/_popups.scss]: done

**Pull request merged**